### PR TITLE
Feature/yarn bootstrap with UI

### PR DESCRIPTION
**Pull request by bendeloyola**
Installed boostrap version 5 using yarn for UI purposes.

command: 
     - yarn add bootstrap@next jquery @popperjs/core
 
Applied Landing page UI 
 
 screenshot:
 
screencapture-localhost-3000-2021-10-13-13_06_29

### CODE REVIEW

**Review by brandomoreno06 — APPROVED** [review]
LGTM.

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -13,3 +13,21 @@
...
+    -moz-user-select: none;
+    user-select: none;
+  }
+
+  @media (min-width: 768px) {
+    .bd-placeholder-img-lg {
+      font-size: 3.5rem;
+    }
+  }
+  //End Landing page style
```
Comment (analyn-cajocson) [inline app/assets/stylesheets/application.scss:33]: add new line

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -15,3 +15,12 @@ require("channels")
...
 // const imagePath = (name) => images(name, true)
+
+require("bootstrap")
+import "../../assets/stylesheets/application";
+document.addEventListener("turbolinks:load", function () {
+    $(function () {
+        $('[data-toggle="tooltip"]').tooltip()
+        $('[data-toggle="popover"]').popover()
+    })
+})
```
Comment (analyn-cajocson) [inline app/javascript/packs/application.js:26]: add new line

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (bendeloyola) [reply app/assets/stylesheets/application.scss]: added new line

**Pull request merged**